catalog/video/video-mpegts-transformer/pom.xml

```diff
@@ -197,7 +197,7 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.79</minimum>
+                                            <minimum>0.59</minimum>
```

Comment:
These have gone way down. If there's a good reason for it, please add a comment above to explain why.